textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/predicates/SelectPredicate.java

```diff
@@ -5,8 +5,8 @@
 /**
  * Object representation of a "SELECT (...)" predicate inside a { @code SelectExtractStatement }.
  * Subclasses have specific fields related to its projection functionalities.
- * SelectPredicate --+ SelectAllPredicate
- *                   + SelectFieldsPredicate
+ * SelectPredicate --+ SelectAllFieldsPredicate
```

Comment:
Now I see the difference between "attribute" and "field": "attribute" refers to part of a schema (e.g., a label), while "field" refers to a value (part of a tuple).

If we all agree on this semantic, it will be good to do a separate PR to make the necessary changes based on this interpretation.  @zuozhiw  do you want to create such a PR and do it some time (when the master is stable)?  @FlavioBayer and I will be the reviewers.